MAUI/Calendar/headers.md

```diff
@@ -72,6 +72,105 @@ this.Calendar.HeaderView = new CalendarHeaderView()
 
 ![Month view Header appearnce Changed in .NET MAUI Calendar.](images/header/maui-month-view-header-appearance-changed.png)
 
+## Header appearance using DataTemplate
+
+You can customize the header appearance by using the [HeaderTemplate]() in the [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html). 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 17" %}
+
+  <Calendar:SfCalendar x:Name="Calendar" 
+                         View="Month">
+        <Calendar:SfCalendar.HeaderTemplate>
+            <DataTemplate>
+                <Grid Background = "#987D9A">
+                    <Label x:Name="label" TextColor="White" HorizontalOptions="Center" VerticalOptions="Start">
+                        <Label.Text>
+                            <MultiBinding StringFormat = "{}{0:MMM dd, yyyy} - {1:MMM dd, yyyy}">
+                                <Binding Path="StartDateRange" />
+                                <Binding Path = "EndDateRange" />
+                            </MultiBinding>
+                        </Label.Text>
+                    </Label>
+                    <Label  HorizontalOptions="Center" VerticalOptions="End" Text="{Binding Text}" TextColor="White" />
+                </Grid>
+            </DataTemplate>
+        </Calendar:SfCalendar.HeaderTemplate>
+ </Calendar:SfCalendar>
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view Header appearance changed using template .NET MAUI Calendar.](images/header/maui-month-view-header-data-template.png)
+
+## Header appearance using DataTemplateSelector
+
+You can customize the header appearance by using the [HeaderTemplate]() in the [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html). The `DataTemplateSelector` can choose a `DataTemplate` at runtime based on the value of a data-bound to Calendar header by using the `HeaderTemplate.` It allows you to choose a different data template for each header, as well as to customize the appearance of a particular header based on certain conditions.
+
```

Comment:
You can customize the header appearance by using the [HeaderTemplate](#) in the [SfCalendar](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html). The `DataTemplateSelector` allows you to choose a `DataTemplate` at runtime based on the value bound to the calendar header. This lets you select a different data template for each header and customize the appearance of a specific header based on certain conditions.

Like this prepare view header

Reply:
Content changed.